.github/workflows/lint.yml

```diff
@@ -94,6 +94,7 @@ jobs:
                      --known-failure app/AttributeAccessToken.h \
                      --known-failure app/CommandHandler.h \
                      --known-failure app/CommandHandlerInterface.h \
+                     --known-failure app/CommandResponseSender.h \
```

Comment:
I assume this is being added to known-failure because it's got the same issues as CommandHandler.h in terms of dependency loops?

Reply:
Yes. Same issue since app depending on im and im (InteractionModelEngine) including this header.